### PR TITLE
Only instantiate handlers once.

### PR DESCRIPTION
**Pull request by jens1o**
fixes https://github.com/UDCBOT/bot/issues/27

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -22,8 +23,13 @@ client.on('message', (msg) => {
     // get the first handler that can handle this message
     for (const handler of messageHandlers) {
         if (handler.canHandle(msg.content)) {
-            const iniatedClass = new handler();
-            iniatedClass.handle(msg, client);
+            if (!instantiatedHandlers.has(handler.HANDLER_ID)) {
+                instantiatedHandlers.set(handler.HANDLER_ID, new handler());
+            }
+
+            let handlerInstance = instantiatedHandlers.get(handler.HANDLER_ID);
```
Comment (micschwarz) [inline src/index.ts]: Let should be const, because it's not reassigned

File: src/message/handler/JQueryMessageHandler.ts [hunk 2]
```diff
@@ -6,6 +6,8 @@ export default class JQueryMessageHandler extends AbstractMessageHandler {
 
     private static readonly MESSAGE_REGEX: RegExp = /jquery/gim;
 
+    public static readonly HANDLER_ID: String = "jquery";
```
Comment (micschwarz) [inline src/message/handler/JQueryMessageHandler.ts]: `"` should be `'`

File: src/message/handler/TeamMessageHandler.ts [hunk 3]
```diff
@@ -6,10 +6,11 @@ export default class TeamMessageHandler extends AbstractMessageHandler {
 
     private static readonly MESSAGE_REGEX: RegExp = new RegExp(`^(${creators.join('|')})`, 'i');
 
+    public static readonly HANDLER_ID: String = "team";
```
Comment (micschwarz) [inline src/message/handler/TeamMessageHandler.ts]: `"` should be `'`

File: src/index.ts (hunk 1 above)
Comment (jens1o) [reply src/index.ts]: used to rust(there are no const there).

File: src/message/handler/JQueryMessageHandler.ts (hunk 2 above)
Comment (jens1o) [reply src/message/handler/JQueryMessageHandler.ts]: used to rust(`'` is only for chars, not strings).

File: src/message/handler/TeamMessageHandler.ts (hunk 3 above)
Comment (jens1o) [reply src/message/handler/TeamMessageHandler.ts]: see https://github.com/UDCBOT/bot/pull/46#discussion_r222707768

**Pull request merged**